modules/custom_operations/user_ie_extensions/ov_extension.cpp

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <openvino/core/extension.hpp>
+#include <openvino/core/op_extension.hpp>
+#include <openvino/frontend/extension.hpp>
+
+#include "sparse_conv.hpp"
+#include "sparse_conv_transpose.hpp"
+#include "complex_mul.hpp"
+#include "calculate_grid.hpp"
+#include "grid_sample.hpp"
+#include "fft.hpp"
+
+// clang-format off
+//! [ov_extension:entry_point]
+OPENVINO_CREATE_EXTENSIONS(
+    std::vector<ov::Extension::Ptr>({
+        // Register operation itself, required to be read from IR
+        std::make_shared<ov::OpExtension<TemplateExtension::SparseConv>>(),
+        std::make_shared<ov::OpExtension<TemplateExtension::SparseConvTranspose>>(),
+        std::make_shared<ov::OpExtension<TemplateExtension::ComplexMultiplication>>(),
+        std::make_shared<ov::OpExtension<TemplateExtension::CalculateGrid>>(),
+        std::make_shared<ov::OpExtension<TemplateExtension::GridSample>>(),
+        std::make_shared<ov::OpExtension<TemplateExtension::FFT>>(),
+
+        // Register operaton mapping, required when converted from framework model format
+        std::make_shared<ov::frontend::OpExtension<TemplateExtension::SparseConv>>(),
+        std::make_shared<ov::frontend::OpExtension<TemplateExtension::SparseConvTranspose>>(),
+        std::make_shared<ov::frontend::OpExtension<TemplateExtension::ComplexMultiplication>>(),
+        std::make_shared<ov::frontend::OpExtension<TemplateExtension::CalculateGrid>>(),
+        std::make_shared<ov::frontend::OpExtension<TemplateExtension::GridSample>>(),
+        std::make_shared<ov::frontend::OpExtension<TemplateExtension::FFT>>(),
```

Comment:
@itikhono, this is what we want to replace by `OPENVINO_FAMEWORK_MAP` macro in the operation definition. Examples in this PR are all eligible for it. Could you refer to the code where usage examples can be found for the macro, and add the description of this macro to the documentation?

@likholat, do you by chance know about this macro? Have you tried to use it?

Reply:
@slyalin Unfortunately I didn't know about this macro, could you please share some usage examples?